docs/index.html

```diff
@@ -64,8 +64,12 @@ <h1 class="sidebar-header">Lifespan filter<span class="sidebar-close"><i class="
               <input type="number" id="to">
             </div>
             <div class="lifespan-controls">
-              <button id="lifespan-pause"><i class="fas fa-pause"></i></button>
-              <button id="lifespan-play"><i class="fas fa-play"></i></button>
+              <p>To play a timelapse, select the starting year using the "From" handle, then specify window size by adjusting the "To" handle. 
+                The specified window will move forward in with the velocity specified below.</p>
+              <button id="lifespan-backward"><i class="fas fa-backward"></i></button>
+              <button id="lifespan-play"><i id="play-icon" class="fas fa-play"></i></button>
+              <label id="vel">x 1</label>
```

Comment:
I think rather than placing this button next to the play/pause button, we can place it on the right end. The reason is that now it is placed next to forwarding button and it is intuitive to move forward but it hurts the intuition of backward.

Reply:
so like,
[<<]    x 1    [>>]                                                                [>]
?
Sorry for ASCIIart :D